### PR TITLE
Create UI for checklist

### PR DESCRIPTION
**Pull request by Kikisek**
Added UI for checklist

### CODE REVIEW

File: src/app/containers/CompanyForm.js [hunk 1]
```diff
@@ -62,6 +76,23 @@ export class CompanyForm extends Component {
 						/>
 					</MuiPickersUtilsProvider>
 				</FormGroup>
+				<FormControl margin="normal" style={{display: "block", textAlign: "left"}}>
```
Comment (susiekims) [inline src/app/containers/CompanyForm.js]: i think after #11 is merged, you can remove this inline style :)

**Review by susiekims — CHANGES_REQUESTED** [review]
Looks great! I think we can get rid of the `text-align: left` after #11 is merged.
Also it looks like the indents might be 4 spaces for these two files?

File: src/app/containers/CompanyForm.js (hunk 1 above)
Comment (Kikisek) [reply src/app/containers/CompanyForm.js]: I think so too!

**Review by susiekims — APPROVED** [review]
Perfect!

**Pull request merged**